### ISSUE TITLE
Plugin work with v50?

### ISSUE THREAD

**Issue by k-brain** [andrewspode/ohnoyoudidnt#1]
I've installed this plugin on multiple machines that are all running Chrome v50+ and it's not doing anything.  Just wondering if this is supposed to still work or if it's no longer compatible.

**Comment by andrewspode** [comment]
Good call. It's detecting crashes but they aren't reloading. I'll look into this.

**Comment by k-brain** [comment]
Thank you! Just starting running Chrome in Kiosk mode on a bunch of PCs for digital signage and a random Aw Snap error sucks.  Tried the --no-sandbox switch as well which some people say works but that has no effect either.

**Comment by andrewspode** [comment]
First of all - big thanks for reporting this, or I never would have caught it.

The way the code used to work, is it sent a no-op code to each tab and if it failed, it simply wouldn't return and I would track how many times it would fail. The downside to this method is that there was a delay as I had to let it fail enough times to decide it was crashed, but also that Google now changed the way it worked!

The new method is that it tries to execute a no-op, and if the tab is crashed it will tell me that the tab is closed. However, this is not a genuine close, as a genuine close also fires an event (that I also capture). So if it says it's closed, but there was no event fired - I can be certain it's crashed.

The upshot is, this now works _and_ it takes at most 1 second to refresh the tab. I just pushed it to the Chrome store, so your extension should update automatically :)
